test/parallel/test-buffer-alloc.js

```diff
@@ -8,6 +8,8 @@ const SlowBuffer = require('buffer').SlowBuffer;
 
 const b = Buffer.allocUnsafe(1024);
 assert.strictEqual(1024, b.length);
+assert.strictEqual(0, b.byteOffset);
+assert.strictEqual(0, b.offset);
```

Comment:
Perhaps you should test `.offset` and `.byteOffset` to be equal to `0` on `Buffer`s created using `Buffer.alloc(number)` and/or `Buffer.allocUnsafeSlow(number)` — those are not pooled.